src/Setup.php

```diff
@@ -415,5 +416,88 @@ public function bcgov_block_theme_wp_headers( $headers ) {
         $headers['Strict-Transport-Security'] = 'max-age=10886400; preload';
         return $headers;
     }
+
+	/**
+     * Registers a Custom Post Type for  "Custom Patterns".
+     *
+     * @since 1.0.8
+	 *
+	 * @return void
+     */
+	public function bcgov_block_theme_register_custom_pattern() {
+
+		/**
+		 * Post Type: Custom Patterns.
+		 */
+
+		$labels = [
+			'name'                     => esc_html__( 'Custom Patterns', 'bcgov-block-theme' ),
```

Comment:
can we not just use `__()` instead of `esc_html__`

Reply:
Yes. Done.